ansible/.gitignore

```diff
@@ -1 +1,3 @@
 *.retry
+sonar-extra-vars.yaml
+sonar-hosts
```

Comment:
My thought process is that people can pull the latest inventory and extra-vars information from Google Drive location without worrying about checking in any sensitive data to the repository.

I can revert this, if you disagree.